android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDatabaseMigration.java

```diff
@@ -14,35 +15,20 @@
      * @param oldVersion The existing database version
      * @param newVersion The database version migration will happen to
      */
+
     @Override
     public void migrate(DynamicRealm dynamicRealm, long oldVersion, long newVersion) {
 
-        // Add schema increments and change methods when updating models
-
-        /* Example migration :
-
         // DynamicRealm exposes an editable schema
-        RealmSchema schema = realm.getSchema();
-
-        // Migrate to version 1: Add a new class.
-        if (oldVersion == 0) {
-            schema.create("Person")
-                .addField("name", String.class)
-                .addField("age", int.class);
-            oldVersion++;
-         }
+        RealmSchema schema = dynamicRealm.getSchema();
+        if(oldVersion == DB_VERSION) {
```

Comment:
I think condition will be false and if block will not run. Because DB_VERSION is now 1 and oldVersion was 0. 

@arp95 @iamareebjamal Please check again. Correct me if i am wrong